data/pc/common/features.json

```diff
@@ -639,6 +557,11 @@
       }
     ]
   },
+  {
+    "name": "itemLoreIsAString",
+    "description": "An item's custom Lore is stored as a string in NBT, in older versions it's a list of strings",
+    "versions": ["1.14_major", "latest"]
```

Comment:
According to the [wiki](https://minecraft.fandom.com/wiki/Player.dat_format#History) Lore was changed to string format in 18w43a